src/openassetio-core/tests/typedefsTest.cpp

```diff
@@ -1,8 +1,10 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2022 The Foundry Visionmongers Ltd
```

Comment:
Unusually not updated copyright

Reply:
I'm now spotting a few of these. Is there a guideline for updating/not updating?